### ISSUE TITLE
Extension mangadex errror

### ISSUE THREAD

**Issue by Wabi25** [tachiyomiorg/tachiyomi#7998]
### Steps to reproduce

Install mangadex extension on the tachiyomi mobile app then add any manga title and refresh or look for update on the title

### Expected behavior

There should be no error shown once refresh or update is done

### Actual behavior

The app shows HTTP Error 503 and HTTP 508

### Tachiyomi version

0.13.6

### Android version

Android 11

### Device

Xiaomi redmi note 9pro

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Seishirou101** [comment]
> If this is an issue with an extension, I should be opening an issue in the extensions repository.

also https://github.com/tachiyomiorg/tachiyomi-extensions/issues/13431
